rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -141,6 +141,13 @@
         return classes;
     }
 
+    @Override
+    public Set<Object> getSingletons() {
```

Comment:
Ok, I see the reasons. Thank you for explanation.
I think it would be good to remove both `getClasses()` and `getSingletons()` sometimes later. Now I consider this as necessary.